### ISSUE TITLE
Error: Situation: Annoying Spells, Condition Script Addon v2.8.1 Game Version 11.0.2.56110

### ISSUE THREAD

**Issue by Nifelhein** [mpstark/DynamicCam#103]
Logged in today and was greeted by the addon reporting that a situation has a problem and that i shoudl report it, so here I am:

Game Version 11.0.2.56110
Addon version: 2.8.1

Situation: Annoying Spells, Condition Script

Error message:
```
Error: Annoying Spells (201), condition, runtime

[string "for _, v in pairs(this.buffs) do..."]:2: attempt to call global 'GetSpellInfo' (a nil value)

```

Let me know if I can give any more information to track down the issue.

**Comment by SoulCookie** [comment]
**+1**

Exact same error message.

**Comment by LudiusMaximus** [comment]
Thanks for reporting. Should be fixed with release 2.8.2.
